package.json

```diff
@@ -40,8 +40,8 @@
     "body parser"
   ],
   "peerDependencies": {
-    "@remix-run/react": "^1.10.0",
-    "@remix-run/server-runtime": "^1.10.0",
+    "@remix-run/react": "^1.11.0",
+    "@remix-run/server-runtime": "^1.11.0",
```

Comment:
Is this required? This will be considered a breaking change

Reply:
Yep, it's needed for `<Await>`.